stores/WindSensors/WindSensorsService.ts

```diff
@@ -0,0 +1,27 @@
+/**
+ * Defines all saga methods to make requests to the WindSensors interface.
+ */
+
+export const WindSensorsService = {
+  *getWindSensors(): Generator<any, any, any> {
+    let isError = false;
+
+    return yield fetch(
+      `${process.env.NEXT_PUBLIC_SERVER_HOST}:${process.env.NEXT_PUBLIC_SERVER_PORT}/api/batteries`,
```

Comment:
Please change this to `/api/wind-sensors`

Reply:
done